examples/uav_dynamics_model.py

```diff
@@ -0,0 +1,123 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+"""
+Example of generating a trajectory for a rotorcraft through a set of coarse waypoints 
+"""
+from prog_models.models.uav_model.uav_model import UAVGen 
+
+import numpy as np
+
+def run_example(): 
+
+    # Example 1: generate trajectory from waypoints and ETAs 
+
+    # Define coarse waypoints: waypoints must be defined with a dictionary of numpy arrays or as columns in a text file 
+    # See documentation for specific information on inputting waypoints 
+    waypoints = {}
+    waypoints['lat_deg'] = np.array([37.09776, 37.09776, 37.09776, 37.09798, 37.09748, 37.09665, 37.09703, 37.09719, 37.09719, 37.09719, 37.09719, 37.09748, 37.09798, 37.09776, 37.09776])
+    waypoints['lon_deg'] = np.array([-76.38631, -76.38629, -76.38629, -76.38589, -76.3848, -76.38569, -76.38658, -76.38628, -76.38628, -76.38628, -76.38628, -76.3848, -76.38589, -76.38629, -76.38629])
+    waypoints['alt_ft'] = np.array([-1.9682394, 164.01995, 164.01995, 164.01995, 164.01995, 164.01995, 164.01995, 164.01995, 0.0, 0.0, 164.01995, 164.01995, 164.01995, 164.01995, 0.0])
+    waypoints['time_unix'] = np.array([1544188336, 1544188358, 1544188360, 1544188377, 1544188394, 1544188411, 1544188428, 1544188496, 1544188539, 1544188584, 1544188601, 1544188635, 1544188652, 1544188672, 1544188692])
+
+    # Define model parameters:
+    params_1 = {
+        'flight_plan': waypoints, # Specify waypoints 
+        'dt': 0.1, # Define time step for model generation; this value is also used for simulate_to functionality 
+        'vehicle_model': 'tarot18', # Define vehicle
+    }
+
+    # Create a model object, define noise
+    uav_1 = UAVGen(**params_1)
+    uav_1.parameters['process_noise'] = 0
+
+    # Define future loading function to return empty InputContainer, since there is no user-defined loading in trajectory generation 
+    def future_loading_1(t, x=None):
+        return uav_1.InputContainer({}) 
```

Comment:
Oh great! I think this is a great change

Reply:
Actually, this doesn't work. I get an error, "simulate_to_threshold() missing 1 required positional argument: 'future_loading_eqn' ". I checked a quick simulation with the ThrownObject, and leaving out future_loading works fine, so I'm wondering if it doesn't work here because we have inputs under the hood. If so, another argument to make some changes for modularity soon. I'll keep debugging